bundles/net.tourbook/src/net/tourbook/database/TourDatabase.java

```diff
@@ -4658,7 +4658,7 @@ private boolean updateDbDesign(final Connection conn, int currentDbVersion, fina
       final String[] buttons = new String[] { IDialogConstants.YES_LABEL, IDialogConstants.NO_LABEL };
 
       if ((new MessageDialog(
-            Display.getDefault().getActiveShell(),
+            splashManager.getShell(),
```

Comment:
@wolfgang-ch The solution i found is to grab the shell of the splash manager. What do you think?

Reply:
Interresting solution :-)

For the slideout + messagebox issue I tried several shells, but none worked only the on_top style

Reply:
What is also interesting is that it was only an issue in Linux, not in Windows !?